projects/packages/paypal-payments/src/paypal-ncps/block.json

```
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "jetpack/paypal-ncps",
+	"version": "0.1.0",
+	"title": "PayPal NCPS",
+	"category": "widgets",
+	"icon": "smiley",
```

Maybe we can use the same icon as for the other block here?

```suggestion
	"icon": "<svg viewBox='0 0 24 24' width='24' height='24' xmlns='http://www.w3.org/2000/svg'><path fill='none' d='M0 0h24v24H0V0z' /><path d='M20 4H4c-1.11 0-1.99.89-1.99 2L2 18c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V6c0-1.11-.89-2-2-2zm0 14H4v-6h16v6zm0-10H4V6h16v2z' /></svg>",
```

---

Good catch! Fixed in [c8b6842](https://github.com/Automattic/jetpack/pull/43932/commits/c8b68429c2ce39038249545c1673ad3421cca384).

---

I updated this again to use the PayPal icon as I was using in the PayPal NCPS dev repo in [6643d49](https://github.com/Automattic/jetpack/pull/43932/commits/6643d49f0c441763664252a3ee14767801f3621e).

---

I think we may want to keep that icon without any fill color, to match the other inserter icons in the editor.

---

Sounds good. I'll change it back. We can just keep the PayPal icon within the block itself.

---

I've switched it back in [ca51ac3](https://github.com/Automattic/jetpack/pull/43932/commits/ca51ac3e4ff3fc4d8cf179605afc2109f79e5f09).